libraries/chain/steem_evaluator.cpp

```diff
@@ -2076,6 +2076,33 @@ namespace golos { namespace chain {
             _db.modify(account, [&](account_object& a) {
                 a.last_account_recovery = now;
             });
+
+            reset_vesting_withdraw(account);
+        }
+
+        void recover_account_evaluator::reset_vesting_withdraw(const account_object& account) {
+            stop_withdraw(account);
+            remove_vesting_routes(account);
+        }
+
+        void golos::chain::recover_account_evaluator::stop_withdraw(const golos::chain::account_object &account) {
+            _db.modify(account, [&](account_object &a) {
```

Comment:
`account_object& a` - good
`account_object &a` - not so :)
And fix same in another places.